.expeditor/verify.pipeline.yml

```diff
@@ -11,40 +11,21 @@ expeditor:
 
 steps:
 
-- label: run-lint-and-specs-ruby-3.0
+- label: run-lint-and-specs-ruby-3.4
   command:
     - .expeditor/run_linux_tests.sh rake
   expeditor:
     executor:
       docker:
-        image: ruby:3.0
+        image: ruby:3.4
 
-- label: run-lint-and-specs-ruby-3.1
```

Comment:
I'd leave the 3.1 compatibility in, unless something breaks in 3.1, which I assume it doesn't.

Reply:
@tpowell-progress yes it did not break any tests so I'm guessing it is fine at least test-wise. ci run from when I had it as >= 3.1 running on 3.1 image earlier https://buildkite.com/chef-oss/chef-mixlib-shellout-main-verify/builds/429)